src/ForkCMS/Component/Module/CopyModuleToOtherLocaleCommand.php

```diff
@@ -6,5 +6,108 @@
 
 abstract class CopyModuleToOtherLocaleCommand implements CopyModuleToOtherLocaleCommandInterface
 {
-    abstract public function copy(Locale $fromLocale, Locale $toLocale): bool;
+    /** @var Locale */
+    private $fromLocale;
+
+    /** @var Locale */
+    private $toLocale;
+
+    /** @var array this is used to be able to convert the old ids to the new ones if used in other places */
+    private $idMap;
+
+    /** @var array this is used to be able to convert the old ids to the new ones if used in other places */
+    private $extraIdMap;
+
+    /** @var string */
+    private $moduleName;
+
+    /**
+     * @var CopyModulesToOtherLocaleResults
```

Comment:
Is there a setting in PHPStorm to allow single-line PHPDoc?
Single line comments should actually use `//` instead of `/* */`...

... What is your opinion?